lib/carrierwave/storage/limelight_file.rb

```diff
@@ -33,7 +33,7 @@ def filename
       end
 
       def media_types
-        ['audio/mpeg', 'audio/wav', 'audio/x-m4a']
+        ['audio/mpeg', 'audio/wav', 'audio/x-m4a', 'audio/x-wav']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.